project/project.yml

```diff
@@ -130,3 +130,12 @@ sites:
         database: pacni
         status: development
         default: false
+    cosicani:
+        name: cosica
```

Comment:
Not sure how important this is but name should be "The Commissioner for Survivors of Institutional Childhood Abuse"

Reply:
Done